CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+3.2.1 (2016-06-16)
+---------------------
+- Fido is fork-safe (reactor is not initialized at fido import time).
```

Comment:
daemonization safe rather. We still don't really support forking i.e. if the program forks later once code has been run, it's not going to work.